web/src/modules/common/containers/__tests__/__snapshots__/LanguageFailure.spec.js.snap

```diff
@@ -61,29 +61,42 @@ exports[`LanguageFailure should match snapshot 1`] = `
           "tunewsThemeColorLight": "rgba(0, 122, 168, 0.4)",
         },
         "dimensions": Object {
-          "headerHeightLarge": 90,
-          "headerHeightSmall": 70,
-          "maxWidth": 1100,
-          "minMaxWidth": "screen and (min-width: 1100px)",
-          "smallViewport": "(max-width: 750px)",
-          "toolbarWidth": 125,
+          "native": Object {
```

Comment:
why is native part of this object?

Reply:
Because currently the whole build config is passed. Same was the case for ios and android before already. We could change this, but I would do it in a separate task.

Reply:
I really think we should structure the buildconfig that only web stuff is accessible in the web project. Maybe this can be done in the other PR which is currently open.

Reply:
I would do this in a separate PR to be honest. We already had this behavior before that ios could access androids build config, I don't really see a difference here.

Reply:
Created an issue: https://issues.tuerantuer.org/browse/IGAPP-364